docs/backend.md

```diff
@@ -48,33 +48,57 @@ Finally, use `git status` and `git diff {config_file.yml}` to reveal and review
 ### Contrib
 I've included some of the non-standard Drupal contrib modules and their use below. To see all the modules in the system review the modules dashboard page located at `/admin/modules`. These modules are maintained by the Drupal community and have regular updates for enhancements and security patches.
 
+#### Admin Toolbar
+Enhances the default Drupal Toolbar by adding drop-down menus for quicker access.
+
 #### Allowed formats
 Adds the ability to restrict wysiwyg editor options per field.
 
 #### Autologout
 Fulfills security requirement to log user out when inactive for a designated period of time.
 
 #### Block content template
-Adds a template hook for `block_content` entities
+Adds a template hook for `block_content` entities.
+
+#### CKEditor 5 Embedded Content
+Allows embedding external content using CKEditor 5.
```

Comment:
Let's update this to `Embedded Content` and drop the CKeditor5 part since we are upgrading.